app/models/series.rb

```diff
@@ -37,6 +37,7 @@ def description_md
   has_many :images, -> { where(parent_id: nil) }, class_name: 'SeriesImage', dependent: :destroy
 
   before_validation :set_app_version, on: :create
+  before_validation :update_account_for_stories, on: :update
```

Comment:
On a related note, if we use `after_commit`, we should make sure that callback will raise an exception: http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#error-handling-in-transaction-callbacks